cli/config/configuration-fc.toml.in

```diff
@@ -136,6 +136,11 @@ block_device_driver = "@DEFBLOCKSTORAGEDRIVER_FC@"
 # command line: intel_iommu=on,iommu=pt
 #enable_iommu = true
 
+# Enable IOMMU_PLATFORM, default false
+# Enabling this will result in the VM device having iommu_platform=on set
+# this option will add iommu_platform=true to the virtio PCI and CCW devices that support it 
+#enable_iommu_platform = true
```

Comment:
Are you sure this is relevant to Firecracker?

Reply:
honestly, I'm not sure. I tested for qemu. It could be removed it if it's not a case firecracker support.

Reply:
Removed it in fc configure.